v2/concordium/types.proto

```diff
@@ -19,6 +19,9 @@ option java_multiple_files = true;
 // Has no effect on code generated on other languages.
 option csharp_namespace = "Concordium.Grpc.V2";
 
+import "kernel.proto";
```

Comment:
Shall we call it `common.proto`. Currently the file includes common types. If we intend to add additional logic that is related to the `kernel of the PLT tokens` then it might be fine naming-wise (just saying that currently the file includes common types and is not related to the PLT kernel).

Reply:
I went with `kernel.proto` because it contains types at the interception between the module and the protocol layer and I imagine there might come more types that will fit in that description. I'm not super attached to `kernel.proto` but I'm not a big fan of `common.proto` as it's not very specific.